packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -43,13 +43,13 @@ export function getPropGetters<
         // The `onTouchStart` event shouldn't trigger the `blur` handler when
         // it's not an interaction with Autocomplete. We detect it with the
         // following heuristics:
-        // - the panel is closed AND there are no running requests
+        // - the panel is closed AND there are no pending requests
         //   (no interaction with the autocomplete, no future state updates)
         // - OR the touched target is the input element (should open the panel)
-        const isNotAutocompleteInteraction =
-          store.getState().isOpen === false && !onInput.isRunning();
+        const isAutocompleteInteraction =
+          store.getState().isOpen === true || !store.pendingRequests.isEmpty();
```

Comment:
To use the same evaluation syntax on both ends.

```suggestion
          store.getState().isOpen || !store.pendingRequests.isEmpty();
```